jbehave-core/src/main/java/org/jbehave/core/configuration/MostUsefulConfiguration.java

```diff
@@ -64,6 +67,7 @@ public MostUsefulConfiguration() {
         useStepdocReporter(new PrintStreamStepdocReporter());
         useParanamer(new NullParanamer());
         useViewGenerator(new FreemarkerViewGenerator());
+        useTableTransformerMonitor(new NullTableTransformerMonitor());
```

Comment:
```suggestion
```

Reply:
> 

According to the current documentation, the such elements should be used in "MostUsefulConfiguration", like "FreemarkerViewGenerator", for example

Or this statement need to be changed in the docs?
https://github.com/jbehave/jbehave-core/blob/91109074e1f8546a5e142b845c8214dbfa4b9e55/distribution/src/site/content/configuration.html#L318-L321

Reply:
I would say that's a bit outdated statement